### PR TITLE
sprinkle some SSL onto http

### PR DESCRIPTION
**Pull request by mmoll**
[ci skip]

### CODE REVIEW

**Comment by mmoll** [comment]
I hope this can go in without Redmine issue :)

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Katello
 
-[![Build Status](http://ci.theforeman.org/buildStatus/icon?job=test_katello)](http://ci.theforeman.org/job/test_katello)
+[![Build Status](https://ci.theforeman.org/buildStatus/icon?job=test_katello)](https://ci.theforeman.org/job/test_katello)
 [![Code Climate](https://codeclimate.com/github/Katello/katello/badges/gpa.svg)](https://codeclimate.com/github/Katello/katello)
 [![Dependency Status](https://gemnasium.com/Katello/katello.svg)](https://gemnasium.com/Katello/katello)
```
Comment (stbenjam) [inline README.md]: While you're in here, would you mind removing this? Gemnasium is gone

Comment (mmoll) [reply README.md]: sure, removed :scissors:

**Comment by stbenjam** [comment]
Thanks @mmoll !

**Pull request merged**